.github/workflows/appengine-deploy.yml

```
@@ -52,3 +56,15 @@ jobs:
       - id: test
         name: Test the deloyed URL
         run: curl --fail "${{ steps.deploy.outputs.url }}"
+        
+      - name: Happo Screenshot Testing
```

```suggestion
      - name: Test Happo screenshot 
```
To follow the naming convention of the file.

---

I like that Sue is encouraging a more descriptive name but I actually prefer "Happo Screenshot Testing" over "Test Happo screenshot"

---

How about "Screenshot test via Happo"?